# mode: 0->PositionCommand | 1->VelocityCommand | 2->stepCommand
uint8 mode
float64[] values
